exovetter/modshift/modshift.py

```diff
@@ -400,8 +400,11 @@ def estimate_scatter(phi_days, flux, phi_pri_days, phi_sec_days,
     # Measure rms of all other points
     idx = ~(idx1 | idx2)
     if not np.any(idx):
-        raise ValueError('RMS calculation failed')
-    rms = np.std(flux[idx])
+        Warning('RMS calculation failed. No points remain. \
```

Comment:
This is not a valid syntax. You need to `import warnings` first, and then use `warnins.warn`.